community/modules/file-system/nfs-server/main.tf

```diff
@@ -70,7 +76,7 @@ resource "google_compute_instance" "compute_instance" {
   boot_disk {
     auto_delete = var.auto_delete_disk
     initialize_params {
-      image = var.image
+      image = data.google_compute_image.compute_image.self_link
```

Comment:
I believe that this is going to change the behavior of the module so that the VM is replaced whenever the family introduces a new image. I think this is, on balance, not desirable. If you look at the vm-instance module there is an example of how to make this work so that it triggers only on changes to user inputs.

Rather than re-invent the wheel, this maybe an opportunity to use vm-instance internally.

Regardless, the behavior should be fixed.

Reply:
Addressed, although I did not switch out google_compute_instance for vm-instance for this.